planning/scenario_planning/common/motion_velocity_optimizer/src/motion_velocity_optimizer.cpp

```diff
@@ -457,32 +495,33 @@ autoware_planning_msgs::Trajectory MotionVelocityOptimizer::optimizeVelocity(
     target_vel, input, input_closest, prev_output, prev_output_closest,
     /* out */ initial_vel, initial_acc);
 
-  autoware_planning_msgs::Trajectory optimized_traj;
+  autoware_planning_msgs::msg::Trajectory optimized_traj;
   bool res = optimizer_->solve(initial_vel, initial_acc, input_closest, input, &optimized_traj);
   if (!res) {
-    // ROS_WARN("[optimizeVelocity] fail to solve optimization.");
+    // RCLCPP_WARN(get_logger(), "[optimizeVelocity] fail to solve optimization.");
```

Comment:
THis looks suspicious, if the solver failed, we just continue?

Reply:
Couldn't fix easily, so I just add the TODO here.

Reply:
That's OK